v2/cmd/parse_log/main.go

```diff
@@ -150,11 +156,29 @@ func run(c *cli.Context) error {
 		return fmt.Errorf("cannot create kafka publisher: %w", err)
 	}
 
+	cowTradeStorage := cowProtocolStorage.New(l, db)
 	// trade log handler
-	tradeLogHandler := handler.NewTradeLogHandler(l, rpcNode, manager, promoteeStorage, parsers, promotionParsers, broadcastTopic, kafkaPublisher)
+	tradeLogHandler := handler.NewTradeLogHandler(l, manager, parsers, broadcastTopic, kafkaPublisher)
+	promoteeHandler := handler.NewPromoteeHandler(l, promoteeStorage, promotionParsers)
+
+	cowTradeParser, err := cowTradesParser.MustNewParser()
+	if err != nil {
+		return fmt.Errorf("cannot create cow trade parser: %w", err)
+	}
+	cowTradeParsers := []iCowParser.TradeParser{
+		cowTradeParser,
+	}
+	cowTransferParser, err := cowTransfersParser.MustNewParser()
+	if err != nil {
+		return fmt.Errorf("cannot create cow transfer parser: %w", err)
+	}
+	cowTransferParsers := []iCowParser.TransferParser{
+		cowTransferParser,
+	}
```

Comment:
```suggestion
	cowTradeParser, err := cowTradesParser.MustNewParser()
	if err != nil {
		return fmt.Errorf("cannot create cow trade parser: %w", err)
	}
	
	cowTransferParser, err := cowTransfersParser.MustNewParser()
	if err != nil {
		return fmt.Errorf("cannot create cow transfer parser: %w", err)
	}
```
Pls keep the code simple, the effort of updating to use multiple parsers is small